apps/www/registry/new-york/ui/timeline.tsx

```diff
@@ -0,0 +1,205 @@
+import React from "react"
+import { VariantProps, cva } from "class-variance-authority"
+import { Check, Circle, X } from "lucide-react"
```

Comment:
not entirely certain, but i think ny variant should use radix-ui/react-icons instead of lucide

Reply:
fixed in https://github.com/shadcn-ui/ui/pull/3374/commits/eabf82bd00a0deae0ee1db479ea92b38e72b2770